docs/docs/tutorial/k-lib.md

```diff
@@ -69,13 +59,19 @@ vendor
 ```
 
 > **Info**: The `vendor/` is the location for external libraries, while `lib/`
-> can be used for your own ones. Check [import paths](/libraries/import-paths) for more information.
+> can be used for your own ones. Check [import paths](/libraries/import-paths)
+> for more information.
 
 ## Aliasing
 
-While you could already use the library by importing `ksonnet.beta.4/k.libsonnet`, this has a drawback: Because the Kubernetes API version is indirectly included in the import name, it makes it impossible to create version agnostic downstream libraries.
+While you could already use the library by importing
+`ksonnet.beta.4/k.libsonnet`, this has a drawback: Because the Kubernetes API
+version is indirectly included in the import name, it makes it impossible to
+create version agnostic downstream libraries.
 
-As a workaround, most libraries expect the correct version of `k.libsonnet` to be importable as a literal `k.libsonnet` (without any package name prefixes). While Jsonnet-bundler won't let you do that, you can alias it by hand:
+As a workaround, most libraries expect the correct version of `k.libsonnet` to
+be importable as a literal `k.libsonnet` (without any package name prefixes).
+While Jsonnet-bundler won't let you do that, you can alias it by hand:
```

Comment:
don't you want to say that `k.libsonnet` is now created for you?

Reply:
Yeah need to fix that